ROADMAP.md

```diff
@@ -11,44 +11,45 @@ OpenEBS follows a lean project management approach by splitting the development
 This table holds a list of the most active roadmap items. These are the features that are currently getting the most active focus and attention within the project.
 
 ## Current
-These are backlog items that are prioritized and planned to be completed within the next major release. While many are planned items, higher priority is given to usability, stability, resilience, Data integrity issues reported by the community.
-
-
-_Note_: OpenEBS follows an aggressive release cadence with a new minor release every 2 months. For the most current plan and status check out the [release project trackers](https://github.com/orgs/openebs/projects?type=classic). This document is reviewed and updated by the maintainers after each major release.
+These are backlog items that are prioritized and planned to be completed within the next major release. While many are planned items, higher priority is given to usability, stability, resilience, data integrity issues reported by the community.
 
+_Note_: OpenEBS follows a release cadence with a new minor release every 2-3 months.
 
 ## Mayastor Roadmap : 2024 Phase-2
 - Forward facing planned release date, Release version numbers and feature priorities are subject to change as the project Maintainers/Leadership/community continuously update and adjust the **Release Feature bundling Strategy** to react to K8s industry movements, trends and our community influence.
 
 |  ID  | Feature name                   | Description and user story                                            | Release, links, tracking issue, GitHub repo                                                   |
 | :--- | :----------------------------- | :--------------------------------------------------------------------- | :------------------------------------------------------------------------------------------ |
-| 1    | Multi-replica volume snapshot and CSI cloning | Able to take consistent snapshots across all available replicas of a volume                                    | Pri 1 /  Rel: (Q1 2024)    |
+| 1    | Multi-replica volume snapshot and CSI cloning | Able to take consistent snapshots across all available replicas of a volume                                    | Pri 1 /  Rel: (Q3 2024)    |
```

Comment:
Do we need to mark this as done now?

Reply:
Ok, will add a status column.

Reply:
Added status to the last column itself, we need to re-structure the columns better in the future. Will take this up in a later PR.